examples/privatelink-access/README.md

```diff
@@ -21,7 +21,7 @@ created so that it can respond to the ENIs created by the EKS control plane:
 
 ```sh
 terraform init
-terraform apply -target=module.create_eni_lambda -target=module.nlb
```

Comment:
doesn't the lambda that listens to create cluster events need to be created before the cluster?

Reply:
ah disregard - missed the part about eventbridge requiring lambda so this ensures the lambda is created before the cluster

Reply:
Yes, the `module.eventbridge` has dependency on the both the `module.create_eni_lambda` and `module.delete_eni_lambda`, so it will run both. Not only that once the lambdas are successfully created, the targets of the rules are updated to point to those lambdas. 

What was happening before was, the lambda was in place and so was the rule but the rule target was not pointing to lambda. It was something that the `module.eventbridge` addresses in totality.